js/private/js_helpers.bzl

```diff
@@ -159,6 +160,16 @@ def gather_runfiles(ctx, sources, data, deps):
 
         deps: list of dependency targets; only transitive runfiles are gather from these targets
 
+        files: a list of files which should be included in runfiles
+
+        copy_files_to_bin: whether to copy files to the bin directory
```

Comment:
Note that this is "files" instead of "data" because it is copying the `gather_runfiles` `files` attribute  (which is normally `ctx.files.data`, but possibly more such as `entry_point`). Where the `gather_runfiles` `data` attribute is `ctx.attr.data` and used to collect providers instead of files.

Any suggestions for better naming so we can potentially keep the `copy_data_to_bin` naming consistent? Maybe `copy_data_files_to_bin` and rename `files` to `data_files`?